dev-context/rest/error-responses.md

```diff
@@ -59,4 +59,6 @@ Therefore, we created a new type `https://api.otto.de/portal/problems/validation
 - [Registered IANA Media Type](https://www.iana.org/assignments/media-types/application/problem+json)
 - [problem+json JAVA](https://github.com/zalando/problem)
 - [problem+json for Spring MVC](https://github.com/zalando/problem-spring-web)
```

Comment:
@stefanschmidt1701 Is this link obsolete with the new links you have added?

Reply:
Hmm... it depends. As far as I've understood the new features of spring-boot 3.x regarding problem-json, the libary from Zalando is no longer needed. I wouldn't use it in a new service. But there are many services in out organization which use this library